spec/models/apartments_spec.rb

```diff
@@ -1,7 +1,91 @@
 require 'rails_helper'
 
 RSpec.describe Apartment, type: :model do
-  it("Apartment model exists") do
-  expect{Apartment.create}.to_not raise_error 
+
+  user = User.where(email: 'test@example.com').first_or_create(password: '12345678', password_confirmation: '12345678')
+
+  it 'can create an apartment' do
+
+    user.apartments.create(
+      street: "string",
+      city: "string",
+      state: "string",
+      manager: "string",
+      email: "string", 
+      price: "string", 
+      bedrooms: 3, 
+      bathrooms: 3, 
+      pets: "string",
+      image: "www.urlhelper.com"
+    )
+    apartment = Apartment.all
+    expect(apartment.length).to eq 1
+  end
+
+  it 'user cannot create an apartment without street' do
+    apartment = user.apartments.create street: '', city: 'New Orleans', state: "The Boot", manager: 'My own Boss', email: 'szaseason@yahoo.com', price: 'unlimited', pets: '2 small dogs.', bedrooms: 7, bathrooms: 7, image: "http//facebook12345", user_id: 'SzaSeason'
```

Comment:
A few things to change on all of your tests is that when you create an instance, you will not want to have anything there.  Meaning, if you are testing for errors on street, street should not be included at all when you create an apartment.  Also, once again, check the data types of each column.  What data type does the schema show for bedrooms, bathrooms, and user_id?